pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -1,12 +1,17 @@
-{ pkgs,  stdenv, fetchurl, unzip, elasticsearch-oss, javaPackages, elk6Version }:
+{ pkgs, lib, stdenv, fetchurl, unzip, javaPackages, elasticsearch }:
+
 
 let
+  esVersion = elasticsearch.version;
+
   esPlugin = a@{
     pluginName,
     installPhase ? ''
       mkdir -p $out/config
       mkdir -p $out/plugins
-      ES_HOME=$out ${elasticsearch-oss}/bin/elasticsearch-plugin install --batch -v file://$src
+      ln -s ${elasticsearch}/lib $out/lib
+      ES_HOME=$out ${elasticsearch}/bin/elasticsearch-plugin install --batch -v file://$src
```

Comment:
I suppose this marks every plugin as unfree?

Reply:
Ahh nvm i forget to commit changed `all-packages.nix`

Reply:
Ok, fixed now